src/sssom/io.py

```diff
@@ -59,7 +60,7 @@ def parse_file(
     output: TextIO,
     input_format: Optional[str] = None,
     metadata_path: Optional[str] = None,
-    prefix_map_mode: Optional[str] = None,
+    merge_mode: Optional[MergeMode] = None,
```

Comment:
All borders have fallen now, but renaming parameters to central methods like this is really annoying for users, but the change makes sense, so lets just break and mend it all now.

Reply:
I have a feeling this one is unused - it is only part of a method that should itself be private. In fact, I would rather remove this option completely and have it automatically chain on all instances.

Reply:
I reverted this and removed the renames